src/pages/DetailPage/index.tsx

```diff
@@ -0,0 +1,61 @@
+import {useEffect} from 'react'
+import { useParams } from "react-router-dom"
+import { useAppDispatch, useAppSelector } from '../../hooks/hooks'
+import { fetchFilmsByID } from '../../store/slices/filmByIdSlice'
+import { ContentWrapper } from "../../components/ContentWrapper"
+
+import Star from '../../assets/img/star.svg'
+import { prepareDate } from '../../utils/prepareDate'
+import { prepareGenre } from '../../utils/prepareGenre'
+
+import "./styles.scss"
+
+
+const DetailPage: React.FC = () => {
+    const {id} = useParams()
+
+    const dispatch = useAppDispatch()
+
+    useEffect(() => {
+        dispatch(fetchFilmsByID(id))
+    },[dispatch,id])
+
+    const {films} = useAppSelector((state) => state.filmByIdSlice)
+
+    return (
+        <ContentWrapper>
+            {Object.keys(films).length > 0 && <div className='detailPage'>
```

Comment:
лучше вынеси в отдельную переменную для лучшего чтения кода
```suggestion
const isFilmExists = Object.keys(films).length > 0
            {isFilmExists && <div className='detailPage'>
```